ProjectSettings/ProjectSettings.asset

```diff
@@ -650,14 +650,14 @@ PlayerSettings:
       HumanInterfaceDevice: False
       InputInjectionBrokered: False
       InternetClient: False
-      InternetClientServer: True
+      InternetClientServer: False
```

Comment:
Please revert this file.

Reply:
The setting is False in Master / Dev Working and This Feature Branch.
What's the reason to put this back to True @keveleigh ?
(no idea how it became true either)

Reply:
Reverting back to what's in Dev Working branch

Reply:
Resolved